src/promptflow/promptflow/_sdk/entities/_flow.py

```diff
@@ -307,13 +315,40 @@ def __call__(self, *args, **kwargs):
         )
         return result.output
 
+    def _convert_multimedia_data_to_base64(self, invoke_result):
```

Comment:
it's a little wired to put them here

Reply:
why not update and directly use `convert_multimedia_data_to_base64`?

Reply:
move those to main.py.